memcached.c

```diff
@@ -3601,6 +3601,31 @@ static void process_watch_command(conn *c, token_t *tokens, const size_t ntokens
     }
 }
 
+static void process_memlimit_command(conn *c, token_t *tokens, const size_t ntokens) {
+    uint32_t memlimit;
+    assert(c != NULL);
```

Comment:
If we're checking for `c != NULL`, why isn't they're also a NULL check for `tokens`?